gammapy/spectrum/models.py

```diff
@@ -43,10 +148,16 @@ class PowerLaw(SpectralModel):
     reference : float, `~astropy.units.Quantity` 
         :math:`E_0`
     """
-    def __init__(self, index, amplitude, reference):
+    DEFAULT_INDEX = 2.3 * u.Unit('')
+    DEFAULT_AMPLITUDE = 2.5 * 1e-12 * u.Unit('cm-2 s-1 TeV-1')
```

Comment:
Why should models have default parameter values?
I think Sherpa does this also, and Astropy doesn't?
Without having thought about it much, I would suggest to not put defaults ... let users be explicit!?